tests/checkpoint/test_checkpoint.py

```diff
@@ -1395,11 +1412,12 @@ def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_when_
     assert result["success"]
 
 
+@pytest.mark.integration
```

Comment:
looks like this method is marked twice

Reply:
@donaldheppner Nice catch -- fixed.  Thanks.